src/device.rs

```diff
@@ -881,12 +881,33 @@ impl FragmentState {
     }
 }
 
+#[repr(C)]
+pub struct PrimitiveState {
+    pub topology: wgt::PrimitiveTopology,
+    pub strip_index_format: IndexFormat,
+    pub front_face: wgt::FrontFace,
+    pub cull_mode: wgt::CullMode,
```

Comment:
this is `Option` now, actually, so if we pick up the latest we'll need to have this working with a local `enum CullMode`, just like we do with `IndexFormat`